src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -575,32 +575,28 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
     }
 
     renderUpgradeLinkDescription() {
-        const { openUpgradePlanModal = () => {}, showNewUpgradeText = false, trackingProps = {} } = this.props;
+        const { openUpgradePlanModal = () => {}, trackingProps = {} } = this.props;
         const { inviteCollabsEmailTracking = {} } = trackingProps;
         const { upgradeLinkProps = {} } = inviteCollabsEmailTracking;
 
         return (
             <div className="upgrade-description">
                 <UpgradeBadge />
-                {showNewUpgradeText ? (
-                    this.renderCollaboratorMessage('external_collab_newcopy_upgrade_cta')
```

Comment:
still using this function to render message for winning treatment code (it was the same copy, different places). Indifferent about keeping the function or moving it more inline but it feels cleaner this way so I left it. lmk what you think.